src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/provider/TaskManagementGetTaskBySearchForCompletablePactTest.java

```diff
@@ -164,7 +165,7 @@ public void getTasksBySearchForCompletableCriteriaWithWarnings() {
                         PermissionTypes.MANAGE,
                         PermissionTypes.REFER
                     ))),
-            "LEGAL_OPERATIONS",
+            RoleCategory.LEGAL_OPERATIONS.name(),
```

Comment:
can we also update the local consumer to include role_category? TaskManagementGetTasksBySearchForCompletableConsumerTest